public/scripts/index.js

```diff
@@ -0,0 +1,36 @@
+const add_button = document.querySelector('.addCandyButton'); 
+const delete_button = document.querySelector('.addCandyButton'); 
+
+
+fetchFunction('/candies')
+.then((res) =>createCard(res));
+
+fetchFunction('/categories')
+.then((res) =>createList('category',res));
+
+fetchFunction('/flavors')
+.then((res) =>createList('flavor',res));
+
```

Comment:
Also here I recommend creating a reusable function to fetch elements instead of repeating the same code.

For example
```js
const fetchElement = (url, cb) => {
    return fetchFunction(url)
        .then((res) => cb(res));
};

// Fetch elements
fetchElement('/candies', createCard);
fetchElement('/categories', createList);
fetchElement('/flavors', createList);
```